package.json

```diff
@@ -32,7 +32,7 @@
     "webworkify": "^1.2.1",
     "yo-yo": "^1.2.2",
     "yo-yoify": "^3.3.0",
-    "ethereum-remix": "0.0.2-alpha.0.0.4"
+    "ethereum-remix": "0.0.2-alpha.0.0.7"
```

Comment:
when you publish to npm, we always have to incr the version (i did not found any other solution). the suffix will not be kept for normal version. This is only used when i have to publish to npm often.